src/style.css

```diff
@@ -1 +1,98 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+body {
+  font-family: Avenir, sans-serif;
```

Comment:
Ensure that the `font-family` includes proper fallback fonts. While `Avenir` is specified, it's important to have a more common fallback like `Arial` or `Helvetica` for better compatibility across different systems.